app/models/flood_risk_engine/enrollment_exemption.rb

```diff
@@ -20,24 +20,28 @@ def self.include_long_dredging?
       includes(:exemption).where(flood_risk_engine_exemptions: { code: Exemption::LONG_DREDGING_CODES }).any?
     end
 
-    def latest_decision
+    def accept_reject_decision_user
+      User.find_by_id(accept_reject_decision_user_id).try(:email)
```

Comment:
`User` doesn't exist in the Engine. If you do this we get into all sorts of dependency issues. This either needs to be done in a presenter in the BO, or we move the `User` object out of the BO and into the Engine.

It is very important that any dependancy only goes one way. The engine MUST NEVER depend on the behaviour of something defined in one of it's host applications. That way madness lies.

Sorry I didn't pick this up yesterday when I had a quick look at the code.